### PR TITLE
Correct typo of FQDN abbreviation

### PR DESCRIPTION
**Pull request by michellemacrh**
Replace typos FDQN with correct abbreviation FQDN

[DDF] Inventory requires FQDN ?

https://issues.redhat.com/browse/AAP-15819

### CODE REVIEW

File: downstream/modules/platform/con-install-scenario-examples.adoc [hunk 1]
```diff
@@ -13,7 +13,7 @@ Red Hat supports a number of installation scenarios for {PlatformNameShort}. Rev
 The database host set to be installed by the installer needs to be a unique host.
 * Do not install {ControllerName} and {HubName} on the same node for versions of {PlatformNameShort} in a production or customer environment.
 This can cause contention issues and heavy resource use.
-* Provide a reachable IP address or fully qualified domain name (FDQN) for the [automationhub] and [automationcontroller] hosts to ensure users can sync and install content from automation hub from a different node. 
+* Provide a reachable IP address or fully qualified domain name (FQDN) for the [automationhub] and [automationcontroller] hosts to ensure users can sync and install content from automation hub from a different node.
```
Comment (EMcWhinn) [inline downstream/modules/platform/con-install-scenario-examples.adoc:16]: For "automation hub" there is an attribute {HubName}, that you can add.

File: downstream/modules/platform/proc-editing-inventory-file-for-updates.adoc [hunk 2]
```diff
@@ -29,7 +29,7 @@ You can use the same `inventory` file from an existing {PlatformNameShort} 2.1 i
 +
 [NOTE]
 ====
-Provide a reachable IP address or fully qualified domain name (FDQN) for the [automationhub] and [automationcontroller] hosts to ensure that users can synchronize and install content from Ansible automation hub from a different node. 
+Provide a reachable IP address or fully qualified domain name (FQDN) for the [automationhub] and [automationcontroller] hosts to ensure that users can synchronize and install content from Ansible automation hub from a different node.
```
Comment (EMcWhinn) [inline downstream/modules/platform/proc-editing-inventory-file-for-updates.adoc:32]: The attribute for "Ansible automation hub" is {HubNameMain}.

File: downstream/modules/platform/ref-guidelines-hosts-groups.adoc [hunk 3]
```diff
@@ -11,7 +11,7 @@
 * If there is an `[automationhub]` group, you must include the variables `automationhub_pg_host` and `automationhub_pg_port`.
 * Add {HubNameMain} information in the `[automationhub]` group
 * Do not install {HubNamemain} and {ControllerName} on the same node.
-* Provide a reachable IP address or fully qualified domain name (FDQN) for the `[automationhub]` and `[automationcontroller]` hosts to ensure that users can synchronize and install content from {HubNameMain} and {ControllerName} from a different node. 
+* Provide a reachable IP address or fully qualified domain name (FQDN) for the `[automationhub]` and `[automationcontroller]` hosts to ensure that users can synchronize and install content from {HubNameMain} and {ControllerName} from a different node.
```
Comment (EMcWhinn) [inline downstream/modules/platform/ref-guidelines-hosts-groups.adoc:14]: I see that [automationhub] and [automationcontroller] have back ticks in this module but not in the previous two, should it be consistent in all?

**Review by EMcWhinn — COMMENTED** [review]
Just minor comments, otherwise LGTM.

**Review by EMcWhinn — APPROVED** [review]
LGTM

**Comment by annibond** [comment]
I also looked this over, good to me also

**Pull request merged**